lexos/helpers/definitions.py

```diff
@@ -5,3 +5,20 @@
 _WORD_BOUNDARY_REGEX_STR = r'\s'
 
 WORD_REGEX = re.compile(_WORD_REGEX_STR, re.UNICODE)
+
+
+def count_phrase_in_text(phrase: str, text: str):
+    """Count how many times the phrase appears in the text
+
+    :param phrase: string containing a phase or word
```

Comment:
I would think a more precise definition of phrases will help future developers, like:
`a sequence of word that is separated by one space between every word` or `a string that ends on a word ending` or something else. 
You need to defining it based on your program, because this is a new terminology in lexos we need to make sure that this term is well-defined and well-documented for both of our developer and user.